src/sentry/models/releasethreshold.py

```diff
@@ -0,0 +1,35 @@
+from django.db import models
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import FlexibleForeignKey, Model, region_silo_only_model
+
+
+@region_silo_only_model
+class ReleaseThreshold(Model):
+    __relocation_scope__ = RelocationScope.Excluded
+
+    class ReleaseThresholdType(models.TextChoices):
+        TOTAL_ERROR_COUNT = "TEC", _("Total Error Count")
+        NEW_ISSUE_COUNT = "NIC", _("New Issue Count")
+        UNHANDLED_ISSUE_COUNT = "UIC", _("Unhandled Issue Count")
+        REGRESSED_ISSUE_COUNT = "RIC", _("Regressed Issue Count")
+        FAILURE_RATE = "FR", _("Failure Rate")
+        CRASH_FREE_SESSION_RATE = "CFSR", _("Crash Free Session Rate")
+        CRASH_FREE_USER_RATE = "CFUR", _("Crash Free User Rate")
+
+    class ValueType(models.TextChoices):
+        PERCENT_OVER = "PO", _("Percent Over")
+        PERCENT_UNDER = "PU", _("Percent Under")
+        ABSOLUTE_OVER = "AO", _("Absolute Over")
+        ABSOLUTE_UNDER = "AU", _("Absolute Under")
+
+    threshold_type = models.CharField(choices=ReleaseThresholdType.choices, max_length=120)
+    value_type = models.CharField(choices=ValueType.choices, max_length=120)
+    value = models.IntegerField()
+    window = models.IntegerField()  # in seconds
```

Comment:
will there be a max on this?

Reply:
I was thinking we'd add that in the serializer at the API later, that way it's easier to enforce, and if we want to modify it, we don't need to do a DB migration, is there a reason to do it here?

Reply:
no, just asking. i also have the same suggestion as above for here if `IntegerField` may become too small in the future

Reply:
`IntegerField` goes up to 2,147,483,647 ([source](https://adamj.eu/tech/2021/05/08/django-check-constraints-limit-range-integerfield/#:~:text=Django's%20built%2Din%20numerical%20fields,application%20reject%20obviously%20wrong%20numbers.))  which is like 68 years - if we monitor a release for 68 years i will come back to this comment and weep

Reply:
nice lol